de.metas.shipper.gateway.spi/src/main/java/de/metas/shipper/gateway/spi/CountryCodeFactory.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * #%L
+ * de.metas.shipper.gateway.spi
+ * %%
+ * Copyright (C) 2019 metas GmbH
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program. If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+
+package de.metas.shipper.gateway.spi;
+
+import com.google.common.collect.ImmutableMap;
+import de.metas.shipper.gateway.spi.model.CountryCode;
+import de.metas.util.Check;
+import lombok.NonNull;
+
+import java.util.Locale;
+
+public class CountryCodeFactory
```

Comment:
* please move this class and also `de.metas.shipper.gateway.spi.model.CountryCode` to the package of `ICountryDAO`
* teo&I *think* you can make this a singleton, so it doesn't have to be constructed each time someone wants to use it